spring-integration-file/src/main/java/org/springframework/integration/file/remote/server/FileServerEvent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.file.remote.server;
+
+import org.springframework.integration.events.IntegrationEvent;
+
+/**
+ * Base class for file server events.
+ *
+ * @author Gary Russell
+ * @since 5.2
+ *
+ */
+@SuppressWarnings("serial")
+public abstract class FileServerEvent extends IntegrationEvent {
+
+	public FileServerEvent(Object source) {
```

Comment:
Do we need to explain in JavaDos what is `source` typically for these type of events?